man/systemd.net-naming-scheme.xml

```diff
@@ -206,6 +206,11 @@
                   <entry>… <constant>v</constant><replaceable>slot</replaceable></entry>
                   <entry>SR-VIO slot number</entry>
                 </row>
+
+                <row>
+                  <entry>… <constant>r</constant><replaceable>slot</replaceable></entry>
```

Comment:
`r` needs to be added to the `<term><varname>ID_NET_NAME_SLOT=<replaceable>…` header above.

Reply:
will fix